week3/api-services/auth_security_system.py

```
@@ -0,0 +1,682 @@
+                )
+                logger.info(f"Created default user: {user_data['username']}")
+            except Exception as e:
+                logger.debug(f"Default user {user_data['username']} may already exist: {e}")
+    
+    def create_user(self, username: str, email: str, password: str, tier: str = "free") -> str:
+        """Create new user account"""
+        user_id = secrets.token_urlsafe(16)
+        password_hash = hashlib.sha256(password.encode()).hexdigest()
+        
+        if tier not in USER_TIERS:
+            raise ValueError(f"Invalid tier: {tier}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)